meshnesh/react-consume-api
# Code changes

# What does this PR do?

This PR will update code to follow eslint and fix action bugs

## How should this be manually tested?

Clone the repo.
Install the packages npm install then run npm run dev to run the app.
Create an account on the app by registering, confirm your email address.
Test the functionality of the app by using it
